imap_data_access/file_validation.py

```diff
@@ -254,11 +252,11 @@ def extract_filename_components(filename: str | Path) -> dict:
         pattern = (
             r"^(?P<mission>imap)_"
             r"(?P<instrument>[^_]+)_"
-            r"(?P<data_level>[^_]+)_"
+            r"(?P<data_level>[^-]+)_"
```

Comment:
should we undo this change based on recent discussion?

Reply:
Yes, I think you're right, this should be underscore. I've updated it now!